charts/beta/vector/templates/aws-config.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: vector-cloudtrail-aws-config
+data:
+  config: |
+    [default]
+    region                  = {{ .Values.vector.sources.cloudtrail.region }}
+    role_arn                = {{ .Values.vector.sources.cloudtrail.cloudtrail_role_arn }}
+    web_identity_token_file = /var/run/secrets/kubernetes.io/serviceaccount/token
+---
```

Comment:
this will break if cloudtrail is not enabled? Can we not render this if cloudtrail is disabled?